xpdacq/beamtimeSetup.py

```diff
@@ -213,8 +213,31 @@ def load_yaml(f, known_uids=None):
     return obj
 
 
-def _end_beamtime(base_dir=None, archive_dir=None, bto=None, usr_confirm='y'):
-    """ funciton to end a beamtime.
+def _end_beamtime(base_dir=None, archive_dir=None, bto=None):
+    """funciton to end a beamtime.
+
+    It makes a remote copy of local directories, ask for user confirm,
```

Comment:
It archives all the user directories on a remote backup, then after asking the
user to confirm it deletes all local user files to leave a clean environment for 
the next user, then terminates the ipython session.